src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssKeysAndValues = sourceString
+    .split(';')
+    .filter((item) => item.trim().length)
+    .reduce((prev, item) => {
+      const index = item.indexOf(':');
+
+      return {
+        ...prev,
+        [item.slice(0, index).trim()]: item.slice(index + 1).trim(),
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Consider formatting the chain of method calls with proper indentation. For example:

const cssKeysAndValues = sourceString
  .split(';')
  .filter((item) => item.trim().length)
  .reduce((prev, item) => {
    // ... reduce logic
  }, {});